src/test/java/pazone/ashot/coordinates/WebDriverCoordsProviderTest.java

```diff
@@ -1,14 +1,15 @@
 package pazone.ashot.coordinates;
 
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebElement;
+
 import static org.junit.jupiter.api.Assertions.assertAll;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-import org.junit.jupiter.api.Test;
-import org.openqa.selenium.Rectangle;
-import org.openqa.selenium.WebElement;
```

Comment:
please keep the original order of imports

Reply:
Sorry, my IDE sometimes does this. Gonna fix it.